library/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="skip_button">ÜBERSPRINGEN</string>
```

Comment:
Using all-caps text in string resources is an anti-pattern.
Should probably be handled by the `TextView`.